visualize-admin/visualization-tool
# feat: Temporal segmentations

This PR allows Temporal dimensions to be used as segmentation fields.

### How to test
1. Go to this [link](https://visualization-tool-git-feat-segment-by-temporal-dimensions-ixt1.vercel.app/en/create/new?cube=https://energy.ld.admin.ch/sfoe/bfe_ogd84_einmalverguetung_fuer_photovoltaikanlagen/9&dataSource=Prod).
2. Change horizontal axis mapping to `Kanton`.
3. Add segmentation by `Jahr der Vergütung`.
4. See that the chart is segmented by this temporal dimension.
5. See that it's possible to filter out particular years and enable interactive filters like in other dimension types.
6. See that when changing chart type to area or line the temporal-segment is disabled, as we need to use this dimension in horizontal axis.
7. You can also repeat this process with other, temporal-enabled cube (e.g. [Bathing water quality](https://visualization-tool-git-feat-segment-by-temporal-dimensions-ixt1.vercel.app/en/create/new?cube=https://environment.ld.admin.ch/foen/ubd0104/6&dataSource=Prod), which has a day dimension).
